// selenium-smart-wait/lib/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.elementTextMatches = exports.elementTextIs = exports.elementTextContains = exports.elementIsVisible = exports.elementIsSelected = exports.elementIsNotVisible = exports.elementIsNotSelected = exports.elementIsEnabled = exports.elementIsDisabled = exports.elementValueIs = exports.elementIsClicked = exports.elementHasBeenSentKeys = exports.elementsCountIs = exports.stalenessOf = undefined;

var _elementsCountIs = require('./elementsCountIs');

Object.defineProperty(exports, 'elementsCountIs', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_elementsCountIs).default;
  }
});

var _elementHasBeenSentKeys = require('./elementHasBeenSentKeys');

Object.defineProperty(exports, 'elementHasBeenSentKeys', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_elementHasBeenSentKeys).default;
  }
});

var _elementIsClicked = require('./elementIsClicked');

Object.defineProperty(exports, 'elementIsClicked', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_elementIsClicked).default;
  }
});

var _elementValueIs = require('./elementValueIs');

Object.defineProperty(exports, 'elementValueIs', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_elementValueIs).default;
  }
});

var _seleniumWebdriver = require('selenium-webdriver');

var _transformElementCondition = require('./transformElementCondition');

var _transformElementCondition2 = _interopRequireDefault(_transformElementCondition);

var _stalenessOf = require('./stalenessOf');

var _stalenessOf2 = _interopRequireDefault(_stalenessOf);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var stalenessOf = exports.stalenessOf = (0, _transformElementCondition2.default)(_stalenessOf2.default);

var elementIsDisabled = exports.elementIsDisabled = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsDisabled);
var elementIsEnabled = exports.elementIsEnabled = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsEnabled);
var elementIsNotSelected = exports.elementIsNotSelected = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsNotSelected);
var elementIsNotVisible = exports.elementIsNotVisible = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsNotVisible);
var elementIsSelected = exports.elementIsSelected = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsSelected);
var elementIsVisible = exports.elementIsVisible = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementIsVisible);
var elementTextContains = exports.elementTextContains = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementTextContains);
var elementTextIs = exports.elementTextIs = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementTextIs);
var elementTextMatches = exports.elementTextMatches = (0, _transformElementCondition2.default)(_seleniumWebdriver.until.elementTextMatches);
